cypress/page/student-registration.page.ts

```diff
@@ -0,0 +1,134 @@
+import * as dayjs from 'dayjs'
+class StudentRegistrationPage {
+    private studentRegistrationPageURL: string;
+    private firstNameInput: string;
+    private lastNameInput: string;
+    private emailInput: string;
+    private genderComponent: string;
+    private mobileInput: string;
+    private dateOfBirthInput: string;
+    private hobbiesComponent: string;
+    private currentAddressInput: string;
+    private stateInput: string;
+    private cityInput: string;
+    private submitButton: string;
+    private Modaltitle: string;
+    private valuesAfterSave: string;
+
+    constructor() {
+        this.studentRegistrationPageURL = "https://demoqa.com/automation-practice-form"
+        this.firstNameInput = "#firstName";
+        this.lastNameInput = "#lastName";
+        this.emailInput = "#userEmail";
+        this.genderComponent = "#genterWrapper";
+        this.mobileInput = "#userNumber";
+        this.dateOfBirthInput = "#dateOfBirthInput";
+        this.hobbiesComponent = "#hobbiesWrapper input+label";
+        this.currentAddressInput = "#currentAddress";
+        this.stateInput = "#state .css-1wy0on6";
+        this.cityInput = "#city .css-1wy0on6";
+        this.submitButton = "#submit";
+        this.Modaltitle = "#example-modal-sizes-title-lg";
+        this.valuesAfterSave = "tr td";
+    }
+
+    public goToStudentRegistrationForm(): void {
+        cy.visit(this.studentRegistrationPageURL);
+    }
+
+    public fillInForm(personalInformation: any = {}): void {
+        cy.get(this.firstNameInput).type(personalInformation.name);
+        cy.get(this.lastNameInput).type(personalInformation.lastName);
+        cy.get(this.emailInput).type(personalInformation.email);
+        this.selectGender(personalInformation.gender);    
+        cy.get(this.dateOfBirthInput).type('{selectall}' + personalInformation.dateOfBirth);
+        cy.get(this.mobileInput).type(personalInformation.mobileNumber);
+        this.selectHobbies(personalInformation.hobbies);
+        cy.get(this.currentAddressInput).type(personalInformation.currentAddress);
+        this.selectState(personalInformation.state);
+        this.selectCity(personalInformation.city);
+        cy.get(this.submitButton).click({force: true});
+    }
+
+    private selectGender (gender: string) {
```

Comment:
delete the space between method's name and parenthesis

Reply:
Resolved!